bav-ipv-stub/src/handlers/callback.ts

```diff
@@ -0,0 +1,17 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+
+export const handler = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  if (event.queryStringParameters?.error != null) {
+    return {
+      statusCode: 200,
+      body: "Authorisation Grant Failed. See logs for details",
```

Comment:
[nit]: Can we just change to be something like "IPV Callback endpoint" so our journey doesn't end on a confusing page..